Gemfile

```diff
@@ -20,7 +20,6 @@ group :development, :test do
 end
 
 group :development do
-  gem "i18n-tasks", "~> 0.9.37"
```

Comment:
@microstudi just to be sure, you're removing this as its not necessary, because its already a dependency of `decidim-dev`, right?

Reply:
yes